### PR TITLE
docs: replace all <a/> tags with <Link/> in stories #1575

### LINKED ISSUE #1575: docs: replace all <a/> tags with <Link/> in stories
[Monday ticket](https://monday.monday.com/boards/3532714909/pulses/5231383026)

- In `preview.js` replace `LinkComponent` with `Link` from `vibe-storybook-components`
- Remove redundant `<a>` props, if needed adjust them with needed `Link` params
image

### PR DESCRIPTION
**Pull request by prakashAditya639**
fixes #1575 
#### Basic

- [x] Used plop (`npm run plop`) to create a new component.
- [x] PR has description.
- [x] New component is functional and uses Hooks.
- [x] Component is written in TypeScript.

#### Style

- [x] Styles are added to `NewComponent.modules.scss` file inside the `NewComponent` folder.
- [x] Component uses CSS Modules.
- [x] Design is compatible with [Monday Design System](https://design.monday.com/).
- [x] Styles are defined using [monday-ui-style](https://github.com/mondaycom/monday-ui-style) tokens
- [x] Displays correctly in all the themes

#### Storybook

- [x] Stories were added to `/src/components/NewComponent/__stories__/NewComponent.stories.mdx` file.
- [x] Stories include all flows of using the component.
- [x] Stories implemented using [vibe-storybook-components](https://github.com/mondaycom/vibe-storybook-components) components and tokens.

#### Tests

- [x] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

#### Accessibility

- [x] Component is accessible.
- [x] Component is compatible with [Vibe Accessibility Guidelines](https://style.monday.com/?path=/docs/foundations-accessibility--page)

### CODE REVIEW

File: .storybook/preview.js
Comment (SergeyRoyt) [inline .storybook/preview.js:1]: Nice 👍

File: .storybook/preview.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ addParameters({
       h2: SectionName,
       h3: Title,
       li: AnchorListItem,
-      a: LinkComponent,
+      a: Link,
       p: Paragraph,
       Tip,
```
Comment (SergeyRoyt) [inline .storybook/preview.js:46]: Let's add also Link here, this way we'd be able to write `<Link` in our mdx files without importing it explicitly from the `vibe-storybook-components`
```suggestion
      Tip,
      Link,
```

File: src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Story, Canvas, Meta } from "@storybook/addon-docs";
+import { Story, Canvas, Meta, Link } from "@storybook/addon-docs";
```
Comment (SergeyRoyt) [inline src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx:1]: Should be a `Link` from `vibe-storybook-components`
Alternatively, if you do the change in `preview.js` you'd be able to not import it at all - it'll replace either your `<a>` in mdx files either your `<Link>

File: src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx [hunk 3]
```diff
@@ -51,9 +51,9 @@ Hook for controlling focus on specific component e.g. Input.
       description={
         <>
           A React{" "}
-          <a href="https://react.dev/reference/react/useRef" target="_blank" rel="noopener noreferrer">
+          <Link to="https://react.dev/reference/react/useRef">
```
Comment (SergeyRoyt) [inline src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx:54]:
```suggestion
          <Link href="https://react.dev/reference/react/useRef" withoutSpacing>
```

Comment (SergeyRoyt) [reply src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx:54]: Please also check if the text size of the link text and other text is the same. If it's bigger than might be needed to add `size={Link.sizes.SMALL}`

**Comment by SergeyRoyt** [comment]
Please, also replace other occurrences of `<a` in the story files 🙏
image

File: src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx (hunk 2 above)
Comment (prakashAditya639) [reply src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx:1]: Thanks for all the suggestion sir it will help me improve a lot.......

**Comment by prakashAditya639** [comment]
Ok sir I will do all that work and report that to you soon sir............

File: src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx (hunk 3 above)
Comment (prakashAditya639) [reply src/hooks/useSetFocus/__stories__/useSetFocus.stories.mdx]: Why you added without spacing in Hyperlink should that be added to all the hrefs..... that are replaced by <Link> tag

**Pull request closed without merging**